src/pages/correspondent-agent-branches/forms/AgentBranchForm.js

```diff
@@ -0,0 +1,125 @@
+// ** MUI Imports
+import { Grid } from '@mui/material'
+import FormShell from 'src/components/Shared/FormShell'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { useForm } from 'src/hooks/form'
+import { RemittanceSettingsRepository } from 'src/repositories/RemittanceRepository'
+import { useState, useEffect, useContext } from 'react'
+import { useInvalidate } from 'src/hooks/resource'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import * as yup from 'yup'
+import toast from 'react-hot-toast'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+
+export default function AgentBranchForm({ _labels, maxAccess, store, setStore, editMode }) {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { recordId } = store
+
+  const [initialValues, setInitialData] = useState({
+    recordId: recordId || null,
+    agentId: '',
+    swiftCode: '',
+    addressId: '',
+    address: ''
+  })
+
+  const invalidate = useInvalidate({
+    endpointId: RemittanceSettingsRepository.CorrespondentAgentBranches.page
+  })
+
+  const { formik } = useForm({
+    maxAccess,
+    initialValues,
+    enableReinitialize: true,
```

Comment:
could wee please insert the initial Values in the useForm directly without using the state ?

Reply:
it is affecting the functionality